client/src/components/geneExpression/menus/geneMenus.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { connect } from "react-redux";
+import {
+  Button,
+  Menu,
+  MenuItem,
+  Popover,
+  Position,
+  Icon,
+  PopoverInteractionKind,
+} from "@blueprintjs/core";
+
+@connect(() => ({}))
+class GenesetMenus extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  handleDeleteGeneFromGeneset = () => {
+    // const { dispatch, geneset } = this.props;
```

Comment:
this (delete action) appears to be unimplemented.  Intentional?

Reply:
Dead code, removed file